tests/functional/spec/language_code.spec.js

```diff
@@ -124,7 +123,6 @@ describe("Language Code", () => {
     $(SummaryPage.submit()).click();
 
     expect(browser.getUrl()).to.contain("thank-you");
-    expect($(ThankYouPage.submissionSuccessfulTitle()).getText()).to.contain("Diolch am gyflwyno eich cyfrifiad");
```

Comment:
These checks are unnecessary and language checks are already done in integration tests.